packages/sdk-cli/src/models/localRelay/launch.test.ts

```diff
@@ -66,23 +66,24 @@ describe('launch', () => {
       // "[log stream] must have an underlying descriptor (file streams do not until the 'open' event has occurred)"
       // Related: https://github.com/nodejs/node-v0.x-archive/issues/4030
       const nodeArgs = ['-e', `console.${consoleMethodName}('${logOutput}')`];
-      subprocess = await launch(nodeArgs, logFile);
+      const subprocessPromise = launch(nodeArgs, logFile);
 
-      // https://nodejs.org/api/child_process.html#event-error
-      subprocess.on('error', (error) => {
-        return done!(error);
-      });
+      // What happened to https://github.com/nodejs/node/issues/1751?
```

Comment:
TODO?

Reply:
More like a note...ChildProcess class should be exposed as per the github issue, but it isn't :/ so I have to resort to constructor.name. Remove?

Reply:
Add a TODO here to revisit, I think it might be old typings.

Reply:
https://github.com/Fitbit/developer-bridge/pull/176/commits/8a45aea402ac49f40b7747dd1dda4f90dd71df16